addons/product_images/wizard/product_get_pic_using_barcode.py

```diff
@@ -20,8 +20,15 @@ class ProductGetPicUsingBarcode(models.TransientModel):
     _description = 'Get Picture with Barcode Number'
 
     @api.model
-    def default_get(self, fields):
-        default = super().default_get(fields)
+    def default_get(self, fields_list):
+        cron_id = self.env.ref('product_images.ir_cron_get_pics_from_google').id
+        date = self.env['ir.cron.trigger'].search([('cron_id', '=', cron_id)]).call_at
+        if date:
+            raise UserError(_("Another task is already scheduled to pick the images on Google "
+                              "on %(year)04d-%(month)02d-%(day)02d at %(hour)02d:%(minute)02d",
+                               year=date.year, month=date.month, day=date.day,
+                               hour=date.hour, minute=date.minute))
```

Comment:
`datetime.date` and `datetime.time` already have acceptable presentations, so you can just use that, possibly blanking the microseconds field for a cleaner output:
```python
>>> print("%s at %s" % (date.date(), date.time().replace(microsecond=0)))
2021-05-18 at 13:12:51
```
although if this is injected into a translated string, I think it would be sensible to follow the date/time formats configured on the user's language.